packages/admin/src/components/bindingFacade/upload/internalComponents/BareFileRepeaterContainer.tsx

```diff
@@ -8,6 +8,7 @@ import { uploadDictionary } from '../uploadDictionary'
 import { FileInput, FileInputPublicProps } from './FileInput'
 import { SingleFilePreview } from './SingleFilePreview'
 import { useNormalizedUploadState } from './useNormalizedUploadState'
+import { useAccessorErrors } from '../../errors'
```

Comment:
Why to keep `useAccessorErrors` hook and not `useFieldErrors` instead too? Maybe unite naming as done with `FieldErrors` replacing `ErrorAccessor`.